modules/elasticsearch/metadata.yaml

```diff
@@ -213,7 +213,32 @@ setup:
 troubleshooting:
   problems:
     list: []
-alerts: []
+alerts:
+  - name: elasticsearch_node_indices_search_time_query
+    link: 
+      https://github.com/netdata/netdata/blob/master/health/health.d/elasticsearch.conf
```

Comment:
I think it is better to have them on the same line (link: _link_)


What link do we need here? To GH HTML page or [raw content](https://raw.githubusercontent.com/netdata/netdata/master/health/health.d/elasticsearch.conf)?

Reply:
agree, let me format the files?

Reply:
Yes, let me format the files. VScode does this sometimes, I see it is doing it for labels further down below. Will add `|` manually there if we don't like how this looks.

I think HTML page, not raw